[2023-10-19T19:04:53.283+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: simple.pokelist manual__2023-10-19T18:59:49.599553+00:00 [queued]>
[2023-10-19T19:04:53.293+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: simple.pokelist manual__2023-10-19T18:59:49.599553+00:00 [queued]>
[2023-10-19T19:04:53.293+0000] {taskinstance.py:1361} INFO - Starting attempt 2 of 2
[2023-10-19T19:04:53.308+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): pokelist> on 2023-10-19 18:59:49.599553+00:00
[2023-10-19T19:04:53.316+0000] {standard_task_runner.py:57} INFO - Started process 562 to run task
[2023-10-19T19:04:53.323+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'simple', 'pokelist', 'manual__2023-10-19T18:59:49.599553+00:00', '--job-id', '40', '--raw', '--subdir', 'DAGS_FOLDER/pokelist.py', '--cfg-path', '/tmp/tmp9e118p10']
[2023-10-19T19:04:53.328+0000] {standard_task_runner.py:85} INFO - Job 40: Subtask pokelist
[2023-10-19T19:04:53.395+0000] {task_command.py:416} INFO - Running <TaskInstance: simple.pokelist manual__2023-10-19T18:59:49.599553+00:00 [running]> on host f04a086795a0
[2023-10-19T19:04:53.483+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@***.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='simple' AIRFLOW_CTX_TASK_ID='pokelist' AIRFLOW_CTX_EXECUTION_DATE='2023-10-19T18:59:49.599553+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-10-19T18:59:49.599553+00:00'
[2023-10-19T19:04:53.486+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-10-19T19:04:53.486+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python3 /opt/***/scripts/pokelist.py']
[2023-10-19T19:04:53.502+0000] {subprocess.py:86} INFO - Output:
[2023-10-19T19:04:53.612+0000] {subprocess.py:93} INFO - Traceback (most recent call last):
[2023-10-19T19:04:53.612+0000] {subprocess.py:93} INFO -   File "/opt/***/scripts/pokelist.py", line 2, in <module>
[2023-10-19T19:04:53.613+0000] {subprocess.py:93} INFO -     from pyspark.sql import SparkSession
[2023-10-19T19:04:53.613+0000] {subprocess.py:93} INFO - ModuleNotFoundError: No module named 'pyspark'
[2023-10-19T19:04:53.615+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2023-10-19T19:04:53.632+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2023-10-19T19:04:53.636+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=simple, task_id=pokelist, execution_date=20231019T185949, start_date=20231019T190453, end_date=20231019T190453
[2023-10-19T19:04:53.648+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 40 for task pokelist (Bash command failed. The command returned a non-zero exit code 1.; 562)
[2023-10-19T19:04:53.693+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-10-19T19:04:53.715+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
